lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/gateway/KeyStoreLoader.java

```diff
@@ -12,11 +12,11 @@
 import java.util.Objects;
 import java.util.UUID;
 
-class KeyStoreLoader {
```

Comment:
My preference would be for the KeyStoreLoader to not be public as it is specific to the experian iiq API and not relevant to the rest of the project. The intention is for every class that is experian-specific to reside in the `gateway` package